### ISSUE TITLE
Removing Batch Normalization 

### ISSUE THREAD

**Issue by andreped** [xiaochus/MobileNetV3#18]
Is there a reason why removing batch normalization (BN) layers from the architecture would be a problem?

I want to use gradient accumulation, and cannot use BNs directly, so I wanted to remove them from the architecture to see if it improves performance. I managed to remove all BNs and tried training a simple binary image classifier.

However, what I am observing is that the model is not learning, and even worse, it seems to converge towards an F1 that is somehow worse than random guess (~0.30 macro F1). If I add back the BNs then it seems to be converging just fine. Any ideas of why that might happen?

I also tried swapping BN with Instance Normalization to no avail... Perhaps it is just a very strange behaviour at the start of training. I could let it iterate for a while, but I doubt there will be much improvement...

I'm using the MobileNetV3-small architecture btw.

I am using `class_weight` and have had no issue with class imbalance in previous trainings with different architectures on the same dataset. However, from looking at the loss, it might be that it converges to always guessing the rarest class, which would explain the poor F1/ACC. Weird behaviour...

**Comment by andreped** [comment]
Oh, just run a new experiment with a different architecture _that did not have batch normalization_ and I observed the same.

So there is likely something wrong on my end and the MobileNetV3-small wo/BN probably works fine.

Closing for now.
